_episodes/06-ignore.md

```diff
@@ -22,7 +22,7 @@ $ touch a.dat b.dat c.dat results/a.out results/b.out
 ~~~
 {: .bash}
 
-and see what Git says:
+and after initializing Git, see what Git says:
```

Comment:
I am saying about `$git init` command before using `$git status`. Nothing to do with web browser.